src/style/components/_cart.scss

```diff
@@ -0,0 +1,10 @@
+.cart {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 15px;
+  &__empty {
```

Comment:
According to BEM methodology **empty** is modifier because it describes the state and must be defined **_empty**

Reply:
Done